docs/book/src/tasks/experimental-features/cluster-class/write-clusterclass.md

```diff
@@ -676,6 +677,102 @@ individually.
 
 </aside>
 
+## JSON patches tips & tricks
+
+JSON patches specification [RFC6902] requires that the target of
+add operation must exist.
+
+As a consequence ClusterClass authors should pay special attention when the following
+conditions apply in order to prevent errors when a patch is applied:
+
+* the patch is a JSON patch
+* the patch tries to `add` a value to a **slice**
```

Comment:
nit: we're mixing Go and JSON here a bit

Up to you but maybe there's a way to make it clearer when we are talking about the slice in an API type and when we are talking about an array in JSON

But maybe it's easier for users if we don't highlight that difference

Reply:
I agree, I changed to using the RFC wording `array` and added at this occurence: `(which is a **slice** in the corresponding go struct)`